src/libraries/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -1,17 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
I don't think you need this.  This package tries not to use reference assemblies, it only added one for ASP.NET's shared framework.

Reply:
If I don't change this then the asset that gets passed in to the consuming net461 app will be flagged as an issue since the task we run to detect if the project dependencies target netsandard is by using metadata on each dependency to see if they depend on either netstandard.dll or System.Runtime.dll. I tried not changing this one, but then my test fails as our SDK will still try to inject the facades here.

Reply:
Won't a `net461` app only get the System.IO.PipeLines src assembly which targets `net461`?  If it's failing, which assembly triggering the facade injection?

Reply:
The task we run unfortunatelly doesn't tell you which assembly triggered it, but I'll do some investagation locally to see if I can figure out why it's needed.

Reply:
you are right, the reason why my test was failing was because when I initially ran it I hadn't add the configuration to the src assembly so it was picking the ns2.0 src implementation and failing the test. I undid the change only for the ref project locally and the test passes just fine so I'll fix this. Great catch.